src/tx/tx.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import type { Account } from '@planetarium/account';
+import { BencodexDictionary, Value, encode } from '@planetarium/bencodex';
+import type { Currency } from '@planetarium/tx';
+import type { SignedTx, UnsignedTx } from '@planetarium/tx/dist/tx';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { createHash } from "node:crypto";
+import esm_bypass_global from 'src/esm_bypass_global';
+
+console.log(esm_bypass_global);
+const { Address, PublicKey } = esm_bypass_global["@planetarium/account"];
+const { encodeSignedTx, signTx } = esm_bypass_global["@planetarium/tx"];
+const { AwsKmsAccount, KMSClient } = esm_bypass_global['@planetarium/account-aws-kms'];
+const encodeCurrency = esm_bypass_global["@planetarium/tx"].encodeCurrency;
+
+const GENESIS_BLOCK_HASH = Buffer.from("4582250d0da33b06779a8475d283d5dd210c683b9b999d74d03fac4f58fa6bce", "hex");
```

Comment:
Is it constant value or sample value for test?

Reply:
It will be constant value for mainnet and other internal testnets (forked from mainnet). But it should be variable when used in a new network with a new genesis block.